app/Http/Controllers/Api/SearchTitleController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Post;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Tools\ResponseTool;
+
+class SearchTitleController extends Controller
```

Comment:
first thing all of it, you don't need create file `SearchTitleController`, because it's not big features. You need make it in PostController because it's still part of it, so make the function there